src/components/AttachResult/Strings.ts

```diff
@@ -0,0 +1,6 @@
+import { Translation, Language } from '../../utils/translation';
```

Comment:
Need to add this file to the side-effects of package.json

Reply:
what does it do exactly ? 🤔

Reply:
That's false

Reply:
To be fairly honest, I'm quite new to the concept of tree shaking and side-effects, so I won't be the best to explains, but I can give you a link to the documentation: https://webpack.js.org/guides/tree-shaking/ 😅